x/datapool/README.md

```diff
@@ -1,43 +1,113 @@
-## Deploy and Register NFT smart contract
+## Deploy and instantiate NFT smart contract
 
+### Setting voter
 ```shell
 TX_FLAG=(--chain-id {your chainID} --gas auto --gas-prices 5umed --gas-adjustment 1.3)
-SENDER=$(panacead keys show {your address or key of sender} -a)
-panacead tx datapool register-contract {your wasm code} --from $SENDER $TX_FLAG -y
+VOTER=$(panacead keys show {your voter} -a)
+panacead tx staking delegate $VALIDATOR 1000000umed --from $VOTER $TX_FLAG -y
```

Comment:
I'm not sure if I understand this comment correctly, this is just for setting voter(To vote, staking is required as you know). You meant that validator can also be a proposer? or voter?

Reply:
Aha. What I mean that validators can also make proposals and vote. 
So I said it would be unnecessary to add a voter.(Because everything is possible with the validator)

Reply:
I got it. But, when validator submit a proposal and self-vote yes to the proposal, the proposal is rejected. So I made a voter by staking some coins.

Reply:
Nice 👍

Reply:
As we said earlier, voting with the validator doesn't seem to be a problem.
Therefore, please review the part where the validator uploads the proposal as it is and votes.

Reply:
I see. Let me update README.md as you said 👍

Reply:
updated in [bf74276](https://github.com/medibloc/panacea-core/pull/303/commits/bf7427611628823f82bf410cab814c43396bd692)